services/pgnotifier/pgnotifier.go

```diff
@@ -335,12 +335,22 @@ func (notifier *PgNotifierT) Publish(jobs []JobPayload, priority int) (ch chan [
 	if err != nil {
 		return
 	}
+	defer func() {
+		if err != nil {
+			if rollbackErr := txn.Rollback(); rollbackErr != nil {
+				pkgLogger.Error(err.Error())
+				pkgLogger.Error(rollbackErr.Error())
+			}
```

Comment:
What happens if rollbackErr is null ? Do we not want to print the `err` in that case ?